--- a/src/Intervention/Image/Gd/Font.php
+++ b/src/Intervention/Image/Gd/Font.php
@@ -229,7 +229,7 @@
             }
 
             // x-position corrections for horizontal alignment
-            switch (strtolower($this->align)) {
+            switch (strtolower($this->align ?? '')) {
                 case 'center':
                     $posx = ceil($posx - ($width / 2));
                     break;
@@ -240,7 +240,7 @@
             }
 
             // y-position corrections for vertical alignment
-            switch (strtolower($this->valign)) {
+            switch (strtolower($this->valign ?? '')) {
                 case 'center':
                 case 'middle':
                     $posy = ceil($posy - ($height / 2));
--- a/src/Intervention/Image/Gd/Commands/BrightnessCommand.php
+++ b/src/Intervention/Image/Gd/Commands/BrightnessCommand.php
@@ -16,6 +16,6 @@
     {
         $level = $this->argument(0)->between(-100, 100)->required()->value();
 
-        return imagefilter($image->getCore(), IMG_FILTER_BRIGHTNESS, ($level * 2.55));
+        return imagefilter($image->getCore(), IMG_FILTER_BRIGHTNESS, (int) round($level * 2.55));
     }
 }
